x-pack/plugins/observability/server/plugin.ts

```diff
@@ -161,11 +162,61 @@ export class ObservabilityPlugin implements Plugin<ObservabilityPluginSetup> {
     const { ruleDataService } = plugins.ruleRegistry;
 
     if (config.unsafe.slo.enabled) {
+      plugins.features.registerKibanaFeature({
+        id: sloFeatureId,
+        name: i18n.translate('xpack.observability.featureRegistry.linkSloTitle', {
+          defaultMessage: 'SLOs',
+        }),
+        order: 1200,
+        category: DEFAULT_APP_CATEGORIES.observability,
+        app: [sloFeatureId, 'kibana'],
+        catalogue: [sloFeatureId, 'observability'],
+        alerting: [SLO_BURN_RATE_RULE_ID],
+        privileges: {
+          all: {
+            app: [sloFeatureId, 'kibana'],
+            catalogue: [sloFeatureId, 'observability'],
+            api: ['slo_write', 'slo_read'],
```

Comment:
we need that for the alert API to work.
```suggestion
            api: ['slo_write', 'slo_read', 'rac'],
```